doc/rules/function_notation/multiline_promoted_properties.rst

```diff
@@ -0,0 +1,77 @@
+======================================
+Rule ``multiline_promoted_properties``
+======================================
+
+Promoted properties must be on separate lines.
+
+Configuration
+-------------
+
+``keep_blank_lines``
+~~~~~~~~~~~~~~~~~~~~
+
+Whether to keep blank lines between properties.
+
+Allowed types: ``bool``
+
+Default value: ``false``
```

Comment:
@kubawerlos , would be good to have documentation example with `keep_blank_lines`

Reply:
With the non-default value it would be not really useful as it would show no changes. Or, do we want to make it `true` by default?

Reply:
imho default should be to properly format the empty lines. if there is phpdoc, it should have a leading empty line between properties. but if there is no doc, no need to keep the line.

this looks wrong to me
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8595/files?diff=unified&w=1#diff-9ab39d90b628a6c9679fd641c7fd271da83074bdb3f0b105023358479d471903R253-R258